### DISCUSSION TITLE
How to handle Indexers with freeleech tokens

### DISCUSSION THREAD

**Discussion by NGDM** [cross-seed/cross-seed#412] [Q&A] [answered]
I'm wondering if anyone has this same issue.
I have some indexers set up in Prowlarr to use any available freeleech tokens when grabbing.
The thing with this app is that it seems to grab torrents to compare them, which means that these tokens are potentially being wasted.

Is there some way to prevent this? Maybe a URL parameter that I can add to the Prowlarr torznab endpoint?
Or I could try duplicating these indexers in Prowlarr and have one freeleech and one non-freeleech…
Wondering how the userbase is handling this?

**Comment by 2fst4u** [comment]
Your second option is probably the best. In reading this topic you just made me think about my own potential wasted tokens, so I might have to steal that idea and implement it myself.

**Reply by 2fst4u** [reply, marked as answer]
I ended up doing this and have duplicated indexers in prowlarr which don't get used by my *arrs (everything disabled in their own special profile, so they still sync to the *arrs but don't do anything). The non freeleech tokens only get used by cross seed and it works well.

**Reply by NGDM** [reply]
I did the same indeed, forgot to post my results, but I agree that this is the best solution for the time being.
An URL flag key/value on the Prowlarr endpoint could be useful, but I guess this would need to be coordinated with the Prowlarr team...
